### DISCUSSION TITLE
Trying to use it in TSX

### DISCUSSION THREAD

**Discussion by navdeep-soni** [RAAbbott/polaris-components#8] [Q&A]
Hi, 

I was looking the the sample code and noticed the below code

import {
  MobileHorizontalDotsIcon,
  LayoutLogoBlockIcon,
  ChevronDownIcon,
  ChevronUpIcon,
  TickIcon,
  CancelIcon
} from '@shopify/polaris-icons';

But some of the icons like MobileHorizontalDotsIcon, TickIcon, CancelIcon are not part of polaris icons which is creating issues in tsx file.
From where does these icons coming from in jsx 

Thanks

**Comment by abdulrafaykhatri** [comment]
Hi Navdeep Soni, these icons are from the latest `@shopify/polaris-icons` library (v8). Please ensure your project is using this version to access these icons. If your version is lower, the icon names might be different.

**Comment by RAAbbott** [comment]
@navdeep-soni I think this was before I updated to use the latest icon library version which had new icon names, and I missed updating some of the text which is why you were getting errors. You shouldn't see these issues anymore on the current components, if you do then let me know
